### ISSUE TITLE
NMT model architecture

### ISSUE THREAD

**Issue by sidthekidder** [bert-nmt/bert-nmt#10]
Is it not possible to use a non-fairseq NMT model for the saved checkpoint? I get a `keyError - state['best_loss']` in checkpoint_utils.py when I try to warmup with an open-nmt transformer model.

**Comment by bert-nmt** [comment]
Yes, because this two different codebase have different saved state_dict. In order to ensure dictionary compatibility during loading, we recommend that you use fairseq checkpoint or discard useless attributes.
